src/checkout-buttons/strategies/braintree/braintree-paypal-button-options.ts

```diff
@@ -10,6 +10,11 @@ export interface BraintreePaypalButtonInitializeOptions {
      */
     shouldProcessPayment?: boolean;
 
+    /**
+     * The ID of a container which the messaging should be inserted.
+     */
+    messagingContainer?: string;
```

Comment:
I think it would be better to use messagingContainerId name. Because we have containerId property

Reply:
Renamed. messageContainerId is optional, when banners are off in  the settings this option will not exist

Reply:
Do you have changes for this one?

Reply:
We recieved 'messagingContainer' from back-end, so in order to rename this variable we need to change it on back-end side.

Reply:
If we receive it from backend, and the backend is controlled by us. I think it's better to make that change now rather than expose it now. Because it will become harder to change in future?

Reply:
Ok, i'll put it into work. Thank you

Reply:
@animesh1987 i believe it's not critical right now. Moreover we have already similar naming in `paypalcommerce` strategy:
https://github.com/bigcommerce/checkout-sdk-js/blob/master/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts#L40
```
const messagingContainer = options.paypalCommerce?.messagingContainer;
``` 
so we made it similar for consistency.
Sure, we could refactor both modules in the future of course when we have more time for it